src/cf-tables.less

```diff
@@ -0,0 +1,484 @@
+// Import external dependencies
+
+@import '../../cf-core/src/cf-core.less';
+
+/* ==========================================================================
+   Capital Framework
+   Table Styling
+   ========================================================================== */
+
+
+
+/* topdoc
+  name: Tables
+  family: cf-tables
+  patterns:
+    - name: Table base style (inherited from cf-core)
+      markup: |
+        <table>
+            <thead>
+                <tr>
+                    <th>Column 1 header</th>
+                    <th>Column 2 header</th>
+                    <th>Column 3 header</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Row 1, column 1</td>
+                    <td>Row 1, column 2</td>
+                    <td class="table__right-align">13</td>
+                </tr>
+                <tr>
+                    <td>Row 2, column 1</td>
+                    <td>Row 2, column 2</td>
+                    <td class="table__right-align">23</td>
+                </tr>
+                <tr>
+                    <td>Row 3, column 1</td>
+                    <td>Row 3, column 2</td>
+                    <td class="table__right-align">33</td>
+                </tr>
+            </tbody>
+        </table>
+      notes:
+        - "The table is not responsive without adding one of the 'small screen'
+            classes below."
```

Comment:
I did that originally by having the default small view be "small stacked" as prescribed by the specification. However, to change to the other small variation required a lot of over-riding, and it made the LESS very messy. I asked @ascott1 for his thoughts, and he suggested that the dev could simply choose which responsive class to use, and thus make the LESS a lot cleaner.

I am open to changing approaches, I just didn't have a better idea at the time. :smile: :octopus::question: